src/main/scala/it/pagopa/interop/purposeprocess/api/impl/PurposeApiServiceImpl.scala

```diff
@@ -166,9 +171,11 @@ final case class PurposeApiServiceImpl(
       uuid           <- id.toFutureUUID
       purpose        <- purposeManagementService.getPurpose(uuid)
       eService       <- catalogManagementService.getEServiceById(purpose.eserviceId)
+      tenant         <- tenantManagementService.getTenant(organizationId)
+      tenantKind     <- tenant.kind.toFuture(TenantKindNotFound(tenant.id))
       authorizedPurpose =
         if (organizationId == purpose.consumerId || organizationId == eService.producerId)
-          purpose.dependencyToApi(isRiskAnalysisFormValid(purpose.riskAnalysisForm))
+          purpose.dependencyToApi(isRiskAnalysisFormValid(purpose.riskAnalysisForm)(tenantKind))
```

Comment:
we need to consider draft vs other states of the purpose:
- DRAFT -> we validate the form and return the result
- non-DRAFT -> the result is always `true`
(of course, with the assumption that a purpose can be moved from DRAFT only if the risk analysis is valid)

This distinction is required because we will have this use case:
- purpose P1 in DRAFT with risk analysis v1.0, which is the latest
- purpose P1 is ACTIVATEd successfully
- risk analysis v2.0 is released
- purpose P1 in ACTIVE state is retrieved and its risk analysis is valid

But if we recalculate the validity based on the latest version, the validity would result as `false`

Reply:
In case of DRAFT purpose we need to return always true or we can try to validate only for schema?